docs/USAGE.md

```diff
@@ -10,7 +10,7 @@ Use these environment variables to configure the tracing library:
 
 | Environment variable | Description | Default |
 |-|-|-|
-| `OTEL_TRACE_CONFIG_FILE` | The path to the configuration file. | `%CurrenctDIR%/datadog.json` |
+| `OTEL_TRACE_CONFIG_FILE` | Loads the json configuration file with configuration values. Overloaded value must be set via Environment variable. | ```{Environment.CurrentDirectory}/datadog.json``` |
```

Comment:
`{Environment.CurrentDirectory}/datadog.json` -> I suggest to use `CurrentWorkingDirectory` or just `WorkingDirectory` https://en.wikipedia.org/wiki/Working_directory